addons/account/tests/test_account_move_line_tax_details.py

```diff
@@ -98,61 +99,61 @@ def test_affect_base_amount_1(self):
         self.assertTaxDetailsValues(tax_details, [
             {
                 'base_line_id': base_lines[0].id,
-                'tax_line_id': tax_lines[2].id,
+                'tax_line_id': tax_lines[3].id,
```

Comment:
Not really, it is because of the change on these lines
https://github.com/odoo/odoo/blob/67add2bab35a8ad664e11429ecd73e4a422e8a01/addons/account/tests/test_account_move_line_tax_details.py#L12-L17

The `sorted` actually didn't do anything because you can't compare recordsets. The `__gt__`, `__lt__` etc for recordset is for comparing sets, not ids.
So yes, it is because of the change you mentioned, but also because of the one I mentioned.